### DISCUSSION TITLE
有没有大佬出一个详细的部署教程

### DISCUSSION THREAD

**Discussion by HughZH** [YunYouJun/valaxy#273] [Q&A]
就是关于部署到GitHub的具体教程

**Comment by YunYouJun** [comment]
GitHub Pages 吗？

参见这里 https://valaxy.site/guide/deploy#github-pages

**Reply by HughZH** [reply]
就是关于上传和更新的问题。
![屏幕截图 2023-10-04 081816](https://github.com/YunYouJun/valaxy/assets/65410989/83bb01b6-c129-42f3-9a89-34e52cad91e8)
之前hexo构架时候，有一个hexo-git部署包，然后很简单的去更新上传文件。就是想问一下大佬，现在valaxy有没有类似的。

**Reply by YunYouJun** [reply]
现在只要正常 `git push` 源代码上去就会自动执行部署，不再需要用户上传构建后的文件了。

单纯 push 代码，可以参考这里。https://www.yunyoujun.cn/posts/how-to-build-your-site

**Reply by YunYouJun** [reply]
后续有计划做一个更加低门槛的客户端 [valaxy-admin](https://github.com/valaxyjs/valaxy-admin)，可以只通过界面操作完成。

**Reply by HughZH** [reply]
大佬np
